Email Not Sent Due To Error

```js
[2024-03-18 10:42:35 UTC] => [ERROR] => Newsletter: TypeError: Cannot read properties of null (reading 'replace')
at Object.decode (/ghosler/node_modules/he/he.js:232:15)
at file:///ghosler/utils/newsletter.js:218:58
at Array.forEach (<anonymous>)
at #injectUrlTracking (file:///ghosler/utils/newsletter.js:218:11)
at async Newsletter.renderTemplate (file:///ghosler/utils/newsletter.js:147:19)
at async Newsletter.send (file:///ghosler/utils/newsletter.js:33:60)
```

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "ghosler",
   "description": "Send newsletter emails to your members, using your own email credentials!",
-  "version": "0.92",
+  "version": "0.93",
   "private": true,
   "main": "app.js",
   "type": "module",

--- a/utils/newsletter.js
+++ b/utils/newsletter.js
@@ -213,9 +213,8 @@ export default class Newsletter {
             renderingData.newsletter.unsubscribeLink,
             renderingData.newsletter.feedbackLikeLink,
             renderingData.newsletter.feedbackDislikeLink,
-            // featuredImage can be null, so we should to filter them.
-            ...renderingData.post.latestPosts.filter(post => post.featuredImage).map(post => post.featuredImage)
-        ].forEach(internalLinks => urlsToExclude.push(he.decode(internalLinks)));
+            ...renderingData.post.latestPosts.map(post => post.featuredImage)
+        ].forEach(linkToTrack => linkToTrack && urlsToExclude.push(he.decode(linkToTrack)));
 
         const trackedLinks = new Set();
         const $ = cheerio.load(renderedPostData);

